diff --git a/src-tauri/gen/android/app/src/main/java/io/github/cloudpilotemu/MainActivity.kt b/src-tauri/gen/android/app/src/main/java/io/github/cloudpilotemu/MainActivity.kt
index 5428306..41b2edd 100644
--- a/src-tauri/gen/android/app/src/main/java/io/github/cloudpilotemu/MainActivity.kt
+++ b/src-tauri/gen/android/app/src/main/java/io/github/cloudpilotemu/MainActivity.kt
@@ -2,10 +2,23 @@ package io.github.cloudpilotemu
 
 import android.os.Bundle
 import androidx.activity.enableEdgeToEdge
+import androidx.core.view.WindowCompat
+import android.webkit.ServiceWorkerController
+import android.webkit.ServiceWorkerClient
+import android.webkit.WebResourceRequest
+import android.webkit.WebResourceResponse
 
 class MainActivity : TauriActivity() {
   override fun onCreate(savedInstanceState: Bundle?) {
     enableEdgeToEdge()
     super.onCreate(savedInstanceState)
+
+    WindowCompat.setDecorFitsSystemWindows(window, true)
+
+    ServiceWorkerController.getInstance().setServiceWorkerClient(object : ServiceWorkerClient() {
+      override fun shouldInterceptRequest(request: WebResourceRequest): WebResourceResponse? {
+          return null
+      }
+    })
   }
 }
